web/source/osk/banner.ts

```diff
@@ -423,10 +423,10 @@ namespace com.keyman.osk {
       let keyman = com.keyman.singleton;
       let manager = this.manager;
 
-      keyman.modelManager['addEventListener']('invalidatesuggestions', manager.invalidateSuggestions);
-      keyman.modelManager['addEventListener']('suggestionsready', manager.updateSuggestions);
-      keyman.modelManager['addEventListener']('tryaccept', manager.tryAccept);
-      keyman.modelManager['addEventListener']('tryrevert', manager.tryRevert);
+      keyman.core.languageProcessor.on('invalidatesuggestions', manager.invalidateSuggestions);
```

Comment:
Does this change the public API for KeymanWeb at all? Are we expecting users to move from `addEventListener` to `on`?

Reply:
Not at all; since predictive text isn't officially a part of KMW (in general, only for the apps it's embedded into), we never publicized this part of the API - events on `modelManager`.